modules/local/get_software_versions.nf

```diff
@@ -8,11 +8,11 @@ process GET_SOFTWARE_VERSIONS {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:'pipeline_info', meta:[:], publish_by_meta:[]) }
 
-    conda (params.enable_conda ? "conda-forge::python=3.8.3" : null)
+    conda (params.enable_conda ? "conda-forge::pyaml==15.8.2" : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/python:3.8.3"
+        container "https://depot.galaxyproject.org/singularity/pyaml%3A15.8.2--py36_0"
```

Comment:
```suggestion
        container "https://depot.galaxyproject.org/singularity/pyaml:15.8.2--py36_0"
```

Reply:
Actually, we need the py**yaml** package which does not have an own biocontainer. 
Would it make sense to use the multiqc container here (as proposed by @mahesh-panchal on slack) which contains a more up-to-date version of py**yaml** than py**aml**? Plus it is downloaded by the pipeline already anyway.

Reply:
Sure. Seeing as we are dumping a file for use by MultiQC in the module too that makes sense.

Reply:
using multiqc container now